Formula/tea.rb

```diff
@@ -0,0 +1,23 @@
+class Tea < Formula
+  desc "Command-line tool to interact with Gitea servers"
+  homepage "https://gitea.com/gitea/tea"
+  url "https://gitea.com/gitea/tea/archive/v0.8.0.tar.gz"
+  sha256 "6c73c0a7b66cdfd1e5a302257d54df06a3a41eb9bdbfeb547966db431ae23b23"
+  license "MIT"
+  head "https://gitea.com/gitea/tea.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    # Remove `if build.head?` on next release.
+    system "make", "vendor" if build.head?
+    system "make"
+    bin.install "tea"
```

Comment:
we should be able to use just `go build`

Reply:
I initially did that, but they pass some stuff to `ldflags`, such as the tea version. Is it still okay to copy those arguments over here?

Reply:
yeah, it is ok, that is what we did in the other go formulae.

Reply:
Done.